src/components/Connection.tsx

```diff
@@ -113,7 +117,7 @@ const Connection: React.FC<ConnectionProps> = ({
     setCustomTime(value);
   };
   //Function to delete all saved files
-  const deletedata = () => {
+  const deletedataall = () => {
```

Comment:
Change the name to deleteAlldata, deletedataall is tough to read and not making sense also